crm_sector/models/crm_lead.py

```diff
@@ -35,3 +37,18 @@ def _lead_create_contact(self, lead, name, is_company, parent_id=False):
             {'sector_id': lead.sector_id.id,
              'secondary_sector_ids': secondary_sector_ids})
         return partner_id
+
+
+class CrmLeadOrm(orm.Model):
```

Comment:
Why this? You can put a method in old API without needing to re-declare the model with `orm.Model`. Any way, you can do it also on new API putting `@api.multi` decorator.